server.go

```diff
@@ -589,66 +598,71 @@ func homeHandler(w http.ResponseWriter, r *http.Request) {
 
 // termsHandler presents the terms.html template
 func termsHandler(w http.ResponseWriter, r *http.Request) {
-	templates := []string{"templates/base.html","templates/terms.html"}
+	templates := []string{"templates/base.html", "templates/terms.html"}
 	t, err := template.ParseFiles(templates...)
 	if err != nil {
 		log.Println("unable to parse terms.html for rendering - %v", err)
 		errHandler(w, r, http.StatusInternalServerError, "internal error parsing templates")
+		return
```

Comment:
without the returns at the end of the error handling, we just keep going. I don't think that was your intended behavior.